src/ser/error.rs

```diff
@@ -30,16 +30,16 @@ pub enum Error {
     ///
     /// To allow serialization of unsigned integers as signed integers, enable the "u2i" feature
     /// flag.
-    UnsupportedUnsignedType,
```

Comment:
Can we update the comment on this variant as well to point to the serde helpers along with `u2i`? Something along the lines of what you added to the display implementation below

Reply:
Good idea, updated. I also updated the docstring and the error message to be a little more general in case users might want to serialize their unsigned integers as doubles or something.